tests/backup/backup_portworx_test.go

```diff
@@ -137,6 +137,7 @@ var _ = Describe("{ReplicaChangeWhileRestore}", func() {
 	labelSelectors := make(map[string]string)
 	params := make(map[string]string)
 	var backupNames []string
+	scName := fmt.Sprintf("replica-sc-%v", time.Now().Unix())
```

Comment:
I would suggest to just declare the `scName` here like `var scName string` and then in the step where you are creating storage class add the time stamp.

```
scName = fmt.Sprintf("replica-sc-%v", time.Now().Unix())
```

This is because ginkgo initialises all the variables at `Describe` level as soon the ginkgo command is run. So if there is another future test case which us creating another sc with name `fmt.Sprintf("replica-sc-%v", time.Now().Unix())` then it would end up in conflict because both of them would have the same time stamp.

Reply:
Done